Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -577,28 +577,41 @@ static RefPtr<CSSValue> positionOffsetValue(const RenderStyle& style, CSSPropert
     return CSSPrimitiveValue::create(CSSValueAuto);
 }
 
+RefPtr<CSSValue> ComputedStyleExtractor::textWrapShorthandValue(const RenderStyle& style) const
```

Comment:
I’m so annoyed that we have to write these functions. I don’t think other browsers support expressing computed style as a `CSSValue` object and all of this code ends up duplicating what the shorthand serializer does, so there are two opportunities to get it wrong. But I guess our editing code depends on this peculiar stuff?

Reply:
I think CSS Typed OM might depend on this code.

Reply:
I'm surprised to hear that. Long term it's really a bummer to have to handle every shorthand twice and also to have to design a non standard CSS value structure for each shorthand when the standards describe the serialization, not objects. At some point worth looking into this to be sure it's valuable.

Reply:
I'll try to see in the future if I can remove this